demo-project/.viz/styles.json

```diff
@@ -0,0 +1,22 @@
+{
+  "companies": {
+    "fill": "red",
+    "color": "white"
+  },
+  "shuttles": {
+    "fill": "green",
+    "color": "white"
+  },
+  "reviews": {
+    "fill": "blue",
+    "color": "white"
+  },
+  "prm_spine_table": {
+    "fill": "orange",
+    "stroke": "blue",
+    "color": "black"
+  },
+  "combine_step": {
```

Comment:
if it's the same func but different namespace; would it work. `data_processing.combine_step`, and `data_science.combine_step`